### PR TITLE
Override Site Executive Form Defaults

### PR DESCRIPTION
**Pull request by martypowell**
These styles override the existing Site Executive default styles. They make it so forms created with site executive match that of the design system.

### CODE REVIEW

File: src/styles/partials/components/_checkbox.scss [hunk 1]
```diff
@@ -1,122 +1,121 @@
...
-		}
-	}
+  .dg_checkbox-label {
+    &::after {
+      position: absolute;
+      top: 8px;
+      left: 11px;
+      top: 3px;
+      left: 6px;
+      content: "\F00C"; /* check - solid - https://fontawesome.com/icons/check?style=solid*/
```
Comment (martypowell) [inline src/styles/partials/components/_checkbox.scss]: This should be moved to a variable, despite this file just being formatted.

**Comment by martypowell** [comment]
@baltimorecounty/front-end-developers whoever grabs this, please let me know and I will make sure they changes are visible on a test page.

**Comment by theilman7** [comment]
Taking a look at this now.

**Comment by theilman7** [comment]
@martypowell In reference to your previous comment on this post, could you send a link to the test page?

File: src/styles/overrides/_se-form.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        @extend .dg_checkbox-input;
+      }
+
+      .seLabelCell {
+        @extend .dg_label;
+        margin-top: 0;
+        margin-bottom: 5px;
+
+        .seRequiredMarker {
+          margin-left: 5px;
```
Comment (theilman7) [inline src/styles/overrides/_se-form.scss]: Should this be set to the variable `$base-spacing-unit`? Same with line 35.

File: src/styles/overrides/_se-form.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      .SEAFGroupVertical,
+      .SEAFLabelVertical {
+        margin-bottom: $default-margin;
+      }
+    }
+
+    .seRequiredMarker {
+      color: $red;
+      font-size: 0;
+      font-weight: 600;
```
Comment (theilman7) [inline src/styles/overrides/_se-form.scss]: I would use the variable` $font-medium`.

File: src/styles/overrides/_se-form.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/overrides/_se-form.scss]: yes

Comment (martypowell) [reply src/styles/overrides/_se-form.scss]: this has been addressed.

File: src/styles/overrides/_se-form.scss (hunk 3 above)
Comment (martypowell) [reply src/styles/overrides/_se-form.scss]: this has been addressed.

File: src/styles/partials/components/_checkbox.scss [hunk 4]
```diff
@@ -1,122 +1,121 @@
...
-	position: relative;
-	min-height: 40px;
-	margin-bottom: 10px;
-	padding-left: 40px;
-	clear: left;
+  line-height: 1.25;
+  display: block;
+  position: relative;
+  min-height: 40px;
+  margin-bottom: 10px;
```
Comment (theilman7) [inline src/styles/partials/components/_checkbox.scss]: This should be set to the variable `$margin-small`.

File: src/styles/partials/components/_checkbox.scss [hunk 5]
```diff
@@ -1,122 +1,121 @@
...
-		left: -2px;
-		width: 44px;
-		height: 44px;
-		margin: 0;
-		opacity: 0;
+  .dg_radio-input,
+  .dg_checkbox-input {
+    cursor: pointer;
+    position: absolute;
+    z-index: 1;
```
Comment (theilman7) [inline src/styles/partials/components/_checkbox.scss]: What is the function of the z-index here? I removed it using inspect element and did not notice any issues.

File: src/styles/partials/components/_checkbox.scss [hunk 6]
```diff
@@ -1,122 +1,121 @@
...
-	min-height: 40px;
-	margin-bottom: 10px;
-	padding-left: 40px;
-	clear: left;
+  line-height: 1.25;
+  display: block;
+  position: relative;
+  min-height: 40px;
+  margin-bottom: 10px;
+  padding-left: 40px;
```
Comment (theilman7) [inline src/styles/partials/components/_checkbox.scss]: This should be set to the variable `$padding-largest`.

File: src/styles/partials/components/_checkbox.scss [hunk 7]
```diff
@@ -1,122 +1,121 @@
...
+    &::before {
+      content: "";
+      -webkit-box-sizing: border-box;
+      box-sizing: border-box;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 40px;
+      height: 40px;
+      border: 2px solid $black;
```
Comment (theilman7) [inline src/styles/partials/components/_checkbox.scss]: Border should be set to border variable `$default-border-width`

File: src/styles/partials/components/_checkbox.scss [hunk 8]
```diff
@@ -1,122 +1,121 @@
...
-			left: 7px;
-			border: 13px solid $black;
-			border-radius: 50%;
-		}
-	}
+    &::after {
+      content: "";
+      top: 7px;
+      left: 7px;
+      border: 13px solid $black;
```
Comment (theilman7) [inline src/styles/partials/components/_checkbox.scss:103]: Question. Do you think the 13px border could or would be used anywhere else on the site? If so, might consider adding a variable for this. If not, fine as is.

File: src/styles/partials/components/_checkbox.scss [hunk 9]
```diff
@@ -1,122 +1,121 @@
...
+  .dg_checkbox-label {
+    &::after {
+      position: absolute;
+      top: 8px;
+      left: 11px;
+      top: 3px;
+      left: 6px;
+      content: "\F00C"; /* check - solid - https://fontawesome.com/icons/check?style=solid*/
+      font-family: $font-awesome-pro;
+      font-size: 28px;
```
Comment (theilman7) [inline src/styles/partials/components/_checkbox.scss]: The variable `$deck-font-size` equals 28px, so that could potentially be used. Thoughts?

**Comment by theilman7** [comment]
Left a few comments and questions. Overall looks good.

**Review by theilman7 — COMMENTED** [review]
Left a few comments and questions. Overall looks good.

File: src/styles/partials/components/_checkbox.scss (hunk 4 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: @theilman7 I did not mean to touch these files...let me get any formatting changse out of here

Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: actually I will just  addresss this now.

Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: addressed

File: src/styles/partials/components/_checkbox.scss (hunk 6 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: addressed

File: src/styles/partials/components/_checkbox.scss (hunk 5 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: good question.

File: src/styles/partials/components/_checkbox.scss (hunk 7 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: addressed

File: src/styles/partials/components/_checkbox.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: addressed

File: src/styles/partials/components/_checkbox.scss (hunk 9 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: yep, addressed, this actually is important in another active pr as well.

File: src/styles/partials/components/_checkbox.scss (hunk 8 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss:103]: this is to make a shape, i think in this case a circle, so no.

Comment (theilman7) [reply src/styles/partials/components/_checkbox.scss:103]: Makes sense, closing conversation.

File: src/styles/partials/components/_checkbox.scss (hunk 5 above)
Comment (martypowell) [reply src/styles/partials/components/_checkbox.scss]: @theilman7 this was not needed. therefore removed, good catch.

**Review by theilman7 — APPROVED** [review]
Looks good to me.

**Pull request merged**